### ISSUE TITLE
Fail run llama2 on ollama0.1.8

### ISSUE THREAD

**Issue by tjlcast** [jmorganca/ollama#1041]
I have same problem.

I reinstall ollama(from 0.1.3 to 0.1.8 ). But when I run `ollama run llama2`, it shows: `Error: llama runner process has terminated`

Memory: 8 GB 1600 MHz DDR3
Graphics: Intel HD Graphics 6000 1536 MB

And `~/.ollama/logs/server.log` like below:
```

[GIN] 2023/11/08 - 18:37:15 | 200 |      27.403µs |       127.0.0.1 | HEAD     "/"
[GIN] 2023/11/08 - 18:37:15 | 200 |    3.545476ms |       127.0.0.1 | POST     "/api/show"
2023/11/08 18:37:15 llama.go:384: starting llama runner
2023/11/08 18:37:15 llama.go:386: error starting the external llama runner: fork/exec /var/folders/1w/bfjzbwc53hbgzsk1spq8f_5w0000gn/T/ollama1055606081/llama.cpp/ggml/build/metal/bin/ollama-runner: bad CPU type in executable
2023/11/08 18:37:15 llama.go:384: starting llama runner
2023/11/08 18:37:15 llama.go:442: waiting for llama runner to start responding
{"timestamp":1699439835,"level":"WARNING","function":"server_params_parse","line":847,"message":"Not compiled with GPU offload support, --n-gpu-layers option will be ignored. See main README.md for information on enabling GPU BLAS support","n_gpu_layers":0}
{"timestamp":1699439835,"level":"INFO","function":"main","line":1191,"message":"build info","build":1009,"commit":"9e232f0"}
{"timestamp":1699439835,"level":"INFO","function":"main","line":1196,"message":"system info","n_threads":2,"total_threads":4,"system_info":"AVX = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | FMA = 0 | NEON = 0 | ARM_FMA = 0 | F16C = 0 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 1 | VSX = 0 | "}
llama.cpp: loading model from /Users/jialtang/.ollama/models/blobs/sha256:8daa9615cce30c259a9555b1cc250d461d1bc69980a274b44d7eda0be78076d8
llama_model_load_internal: format     = ggjt v3 (latest)
llama_model_load_internal: n_vocab    = 32000
llama_model_load_internal: n_ctx      = 2048
llama_model_load_internal: n_embd     = 4096
llama_model_load_internal: n_mult     = 256
llama_model_load_internal: n_head     = 32
llama_model_load_internal: n_head_kv  = 32
llama_model_load_internal: n_layer    = 32
llama_model_load_internal: n_rot      = 128
llama_model_load_internal: n_gqa      = 1
llama_model_load_internal: rnorm_eps  = 5.0e-06
llama_model_load_internal: n_ff       = 11008
llama_model_load_internal: freq_base  = 10000.0
llama_model_load_internal: freq_scale = 1
llama_model_load_internal: ftype      = 2 (mostly Q4_0)
llama_model_load_internal: model size = 7B
llama_model_load_internal: ggml ctx size =    0.08 MB
llama_model_load_internal: mem required  = 3615.73 MB (+ 1024.00 MB per state)
llama_new_context_with_model: kv self size  = 1024.00 MB
llama_new_context_with_model: compute buffer total size =  153.35 MB
2023/11/08 18:37:15 llama.go:399: signal: segmentation fault
2023/11/08 18:37:15 llama.go:407: error starting llama runner: llama runner process has terminated
2023/11/08 18:37:15 llama.go:473: llama runner stopped successfully
```

But before reinstalling I can do ollama run llama2 in ollama(0.1.3)

So how can I fix it?

**Comment by xyproto** [comment]
Which OS is this on?

**Comment by tjlcast** [comment]
Hi, 
MacOS Catalina
version 10.15.7

**Comment by jpmcarvalho** [comment]
I have the same issue in my macOS. Did you solve it?

**Comment by tjlcast** [comment]
@jpmcarvalho  
Hi. I try to reinstall the ollama(0.1.7). It worked.

**Comment by tjlcast** [comment]
And my machine is:
```
Processor: IntelCorei7 
Memory: 8GB 
Graphic: Intel HD Graphics 6000 1536 MB
```

**Comment by jmorganca** [comment]
Hi folks, I'm so sorry you hit this bug on Intel Macs. This should be fixed as of [`41434a7`](https://github.com/jmorganca/ollama/commit/41434a7cdcf33918ae2d37eb23d819ef7361e843). A new release should be out early tomorrow but in the meantime you can try the prerelease version here https://github.com/jmorganca/ollama/releases/tag/v0.1.10. Again thanks for creating an issue and your patience!
